### DISCUSSION TITLE
3rd party BLE device

### DISCUSSION THREAD

**Discussion by MutterOberin** [pybricks/support#986] [Q&A]
Hi guys! 

I am wondering if it is possible to get communication between a robot inventor hub and a 3rd party BLE device. For example the arduino nano 33 ble. The arduino should be seen as a sensor and allow transmitting information into the hub. Would such a thing be possible?

**Comment by laurensvalk** [comment]
Yes, see https://pybricks.com/projects/tutorials/wireless/hub-to-device/pc-communication/. Instead of using a PC as in this example, see if you can make your Arduino send the same messages (Nordic UART service).

**Reply by MutterOberin** [reply]
Thank you very much for this fast answer! Looks promising and I will dig into a bit!

**Reply by laurensvalk** [reply]
Don't forget to let us know if you've made something cool :smile:

**Reply by MutterOberin** [reply]
Oh of course!!
